src/vs/workbench/browser/window.ts

```diff
@@ -101,7 +101,7 @@ export abstract class BaseWindow extends Disposable {
 
 	//#region timeout handling in multi-window applications
 
-	private enableMultiWindowAwareTimeout(targetWindow: Window, dom = { getWindowsCount, getWindows }): void {
+	protected enableMultiWindowAwareTimeout(targetWindow: Window, dom = { getWindowsCount, getWindows }): void {
```

Comment:
This was needed to avoid multiple calls to `defineProperty` of `vscodeOriginalSetTimeout` on the same window instance.